tests/foreman/ui/test_activationkey.py

```diff
@@ -206,6 +207,94 @@ def test_positive_create_with_envs(self):
                     )
                     self.assertIsNotNone(self.activationkey.search(name))
 
+    @run_only_on('sat')
+    @tier2
+    @upgrade
+    def test_positive_access_non_admin_user(self):
+        """Access activation key that has specific name and assigned
+        environment by user that has filter configured for that specific
+        activation key
+
+        :id: 358a22d1-d576-475a-b90c-98e90a2ed1a9
+
+        :expectedresults: Only expected activation key can be accessed by new
+            non admin user
+
+        :BZ: 1463813
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        envs_list = ['STAGING', 'DEV', 'IT', 'UAT', 'PROD']
+        for name in envs_list:
+            entities.LifecycleEnvironment(name=name, organization=org).create()
+        with Session(self) as session:
```

Comment:
is there a specific reason to make test setup with UI?

Reply:
actually it isn't performed in UI or you like me move CV section out of session?
I can do it